lib/brcobranca.rb

```diff
@@ -117,9 +117,14 @@ module Template
   module Retorno
     autoload :Base,            'brcobranca/retorno/base'
     autoload :RetornoCbr643,   'brcobranca/retorno/retorno_cbr643'
-    autoload :RetornoCnab240,  'brcobranca/retorno/retorno_cnab240'
+    autoload :RetornoCnab240,  'brcobranca/retorno/retorno_cnab240' # DEPRECATED
     autoload :RetornoCnab400,  'brcobranca/retorno/retorno_cnab400' # DEPRECATED
 
+    module Cnab240
+      autoload :Base,  'brcobranca/retorno/cnab240/base'
+      autoload :Caixa, 'brcobranca/retorno/cnab240/caixa'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.